packages/std/src/conversion.rs

```diff
@@ -12,7 +12,6 @@ pub fn force_to_u32(input: usize) -> u32 {
     }
     #[cfg(not(target_pointer_width = "32"))]
     {
-        use std::convert::TryInto;
```

Comment:
Nice.

What about the `FromIterator` imports in some of the test code and many of the TryInto imports?

Reply:
Hm, good point. One I removed was pointed out by clippy, I'll check it out further.